src/pretix/plugins/sendmail/views.py

```diff
@@ -142,5 +148,6 @@ def get_context_data(self, **kwargs):
             log.pdata['sendto'] = [
                 status[s] for s in log.pdata['sendto']
             ]
+            log.pdata['mailid'] = log.object_id
```

Comment:
You can just access ``log.object_id`` in the template, so there's not really a need for this.